tests/components/eheimdigital/test_light.py

```diff
@@ -127,15 +128,27 @@ async def test_turn_off(
     )
     await hass.async_block_till_done()
 
+    classic_led_ctrl_mock.turn_off.side_effect = EheimDigitalClientError
+
+    with pytest.raises(HomeAssistantError):
```

Comment:
Should we match on the exception string?

Reply:
Honestly, that would have my preference too... It add validation that you are actually catching the expected error.

Reply:
I have added a check :)

Reply:
But as there is nothing I could match on, I used the exc_info